po/fr.po

```diff
@@ -839,18 +840,16 @@ msgid "unknown (bug)"
 msgstr "inconnu (bug)"
 
 #: src/inoreader_urlreader.cpp:34
-#, fuzzy
 msgid "Broadcast items"
-msgstr "Déjà au dernier élément."
+msgstr "Éléments partagés"
 
```

Comment:
I am not a user of Inoreader, but after a quick look at it, it seems that broadcasted items and shared items are the same think, is this correct ?
If so, I move the litteral translation of "shared items", previously on line 849 to this line

Reply:
I don't use Inoreader either, though I did register an account to get an API key. From what I gather, you're right and "broadcast items" in Inoreader is semantically the same as "shared items" in The Old Reader or FeedHQ.